src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageBuildAction.java

```diff
@@ -86,6 +87,7 @@ public final class CoverageBuildAction extends BuildAction<Node> implements Stap
     /** The coverages filtered by modified lines of the associated change request. */
     private final List<? extends Value> modifiedLinesCoverage;
 
+    @SuppressFBWarnings("SE_BAD_FIELD")
```

Comment:
Please add a explanation for each suppression like:
@SuppressFBWarnings(value="SE_BAD_FIELD", justification = "xxx")